exporters/ostream/test/ostream_log_test.cc

```diff
@@ -150,7 +150,7 @@ TEST(OStreamLogExporter, LogWithStringAttributesToCerr)
   auto record = exporter->MakeRecordable();
 
   // Set resources for this log record only of type <string, string>
-  record->SetResource("key1", "val1");
+  record->SetResource("key1", std::string("val1"));
```

Comment:
Could we provide an explicit overload to `SetResource` with `const char *` to avoid potential future error?